docs/source/get_started.rst

```diff
@@ -11,7 +11,7 @@ Installation
 
 ``sktime`` currently supports:
 
-* environments with python version 3.6, 3.7, or 3.8.
+* environments with python version 3.6, 3.7, 3.8, or 3.9.
```

Comment:
thanks!
3.6 is no longer supported, so that should actually be removed.
Thanks for spotting!

Reply:
oh okay, I will do this small fix!